packages/@sanity/types/src/schema/definition/type/document.ts

```diff
@@ -28,4 +28,9 @@ export interface DocumentDefinition extends Omit<ObjectDefinition, 'type'> {
   __experimental_search?: {path: string; weight: number; mapWith?: string}[]
   /** @alpha */
   __experimental_omnisearch_visibility?: boolean
+  /**
+   * Hides the document title introduced at the document pane
+   * @alpha
+   * */
+  __experimental_hide_form_title?: boolean
```

Comment:
```suggestion
  /**
   * Determines whether the large preview title is displayed in the document pane form
   * @alpha
   */
  __experimental_form_preview_title?: boolean
```

I would suggest renaming this to `__experimental_form_preview_title` to be a little more explicit – underscoring that this is a _preview_ title, which is shared across other previews too.

It also require users to set this to `false` to hide it, which I think reads a little better. WDYT?

Reply:
Sounds good! Updated